test/verify/check-ovirt

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2016 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+
+import os
+
+# The oVirt image has following content
+# DC: "Default"
+# Cluster: "Default"
+# Host: "host" running in the image with plaintext communication to engine
+# VM: "VM" Blank-based VM with network boot 
+#
+
+WAIT_SCRIPT = """
+set -ex
+for x in $(seq 1 60); do
+    if [ "$(curl -s -k -u admin@internal:engine https://%(addr)s/ovirt-engine/api | xmllint --xpath "string(/api/product_info/name)" -)" = "oVirt Engine" ]; then
+        break
+    else
+        sleep 10
+    fi
+done
+"""
+
+
+@skipImage("No oVirt release", "continuous-atomic", "debian-stable", "debian-testing", "fedora-24", "fedora-25", "fedora-i386", "fedora-atomic", "fedora-testing", "ubuntu-1604", "ubuntu-stable")
+class TestOVirtMachines(MachineCase):
+    provision = {
+        "0": { "address": "10.111.113.1/20" },
+        "ovirt": { "image": "ovirt", "memory_mb": 4096}
+    }
+
+    def setUp(self):
+        MachineCase.setUp(self)
+        self.ovirt = self.machines['ovirt']
+        m = self.machine
+
+        # Wait for the web service to be accessible
+        args = { "addr": "10.111.113.5" }
+        m.execute(script=WAIT_SCRIPT % args)
+
+        vmid = m.execute("curl -s -k -u admin@internal:engine -H Content-type:application/xml https://10.111.113.5/ovirt-engine/api/vms/ | xmllint --xpath \"string(/vms/vm[name=\'VM\']/@id)\" -")
+	self.assertNotEqual(vmid, "")
+        m.execute("curl -s -k -u admin@internal:engine -H Content-type:application/xml -X POST -d \"<action/>\"  https://10.111.113.5/ovirt-engine/api/vms/" + vmid + "/start")
+
+    def testBasic(self):
+        b = self.browser
+
+        self.login_and_go("/machines")
```

Comment:
yes, it seems to be set up correctly and working manually, but it does require a change in the plugin as now the test machine needs to "talk" to the remote ovirt instance. Normally the deployment is always local - cockpit runs on each ovirt's host and only talks to local libvirt. 
It's a small TODO to make the libvirt URI configurable (same for regular cockpit-machines, see https://github.com/cockpit-project/cockpit/blob/83ac95f549c4ac6e18e445f41012a25e879c9bc8/pkg/machines/config.es6#L23)